exporters/etw/README.md

```diff
@@ -0,0 +1,201 @@
+# Getting Started with OpenTelemetry C++ SDK and ETW exporter on Windows
+
+Event Tracing for Windows (ETW) is an efficient kernel-level tracing facility that lets you log kernel or
+application-defined events to a log file. You can consume the events in real time or from a log file and
+use them to debug an application or to determine where performance issues are occurring in the application.
+
+OpenTelemetry C++ SDK ETW exporter allows the code instrumented using OpenTelemetry API to forward events
+to out-of-process ETW listener, for subsequent data recording or forwarding to alternate pipelines and
+flows. Windows Event Tracing infrastructure is available to any vendor or application being deployed on
+Windows.
+
+## API support
+
+These are the features planned to be supported by ETW exporter:
+
+- [x] OpenTelemetry Tracing API and SDK headers are **stable** and moving towards GA.
+- [ ] OpenTelemetry Logging API is work-in-progress, pending implementation of [Latest Logging API spec here](https://github.com/open-telemetry/oteps/pull/150)
+- [ ] OpenTelemetry Metrics API is not implemented yet.
+
+Implementation of OpenTelemetry C++ SDK ETW exporter on Windows OS is `header only` :
```

Comment:
nit:
```suggestion
Implementation of OpenTelemetry C++ SDK ETW exporter on Windows OS is `header only`:
```

Reply:
Do we need space before colon in text, I thought no.. `In modern English-language printing, no space is placed before a colon and a single space is placed after it.`.. Since this is end-of-line, there's no space, but end-of-line instead.

Reply:
I'm marking it as `Resolved` / `Won't Fix` since this proposal is not gonna be consistent with modern English-language printing rules :)